streaming/rrdpush.c

```diff
@@ -79,6 +79,18 @@ int rrdpush_init() {
         default_rrdpush_enabled = 0;
     }
 
+#ifdef ENABLE_HTTPS
```

Comment:
This is still not working quite as it should

> streaming sockets (not HTTP protocol)
Protocol parameter in bind to is ^SSL=force - Ignore and do the previous, *until netdata slaves can actually use SSL*. After we implement that, only accept encrypted connections.

Essentially, when I tell the master server SSL=force, it should deny the streaming connection. I did a `bind to = *^SSL=force` and it accepted an unencrypted connection.

In other words:
Default for tcp sockets except streaming = force
Default for streaming = optional
Override the default if the user provides ^SSL=....

Reply:
It no longer accepts the unencrypted connection, but the log messages are misleading:
On the master, I don't get any messages. I should be seeing at least a connection refused or something.
On the slave, I see `STREAM fedora [send to 192.168.122.1:19999]: server is not replying properly (is it a netdata?).`

Reply:
It will also be in the next pull, thanks!

Reply:
Comment here when it's ready for testing. I tried again and didn't see a difference.

Reply:
Sorry, I changed here, but I did not execute a pull, because I am finishing the redirect, it will be ready after the Daily Standup.

Reply:
I realized the commit with the reduction of the cipher numbers given by the Netdata and the installer directory, I am returning for the latest bug.